sgl-kernel/csrc/moe/moe_fused_gate.cu

```diff
@@ -39,7 +39,9 @@ __device__ inline bool cmp_eq(const T& a, const T& b) {
 // Fixed constants common to both dynamic and static template versions:
 static constexpr int WARP_SIZE = 32;
 static constexpr int WARPS_PER_CTA = 6;
-static constexpr int MAX_VPT = 32;  // maximum VPT we support, > params.VPT = num_expert / num_expert_group
+// DeepSeek V3/R1: num_experts = 256, n_group = 8
+// Kimi K2:        num_experts = 384, n_group =1
+static constexpr int MAX_VPT = 384;  // maximum VPT we support, > params.VPT = num_expert / num_expert_group
```

Comment:
@ispobock I feel that directly changing MAX_VPT to 384 in this place, without any register reuse strategy, will cause this kernel to experience register overflow. The modification here is extremely sensitive. I personally believe the closest implementation to what we want currently is this change: https://github.com/sgl-project/sglang/pull/6946. This uses a tiling approach for register reuse and also performs data prefetching between tiles. With minor modifications, it should support the Kimi V2 (384) case here. I'll push this forward as soon as possible.